dev-docs/publisher-api-reference/setConfig.md

```diff
@@ -824,6 +825,19 @@ config.setConfig({
 });
 ```
 
+##### Details on the allowSendAllBidsTargetingKeys setting
+
+The `allowSendAllBidsTargetingKeys` is similar to `allowTargetingKeys` except it limits any default bidder specific keys sent to the adserver when sendAllBids is enabled. Any default bidder specific keys that do not match the mask will not be sent to the adserver. This setting can be helpful if you find that your default Prebid.js implementation is sending key values that your adserver isn't configured to process; extraneous key values may lead to the ad server request being truncated, which can cause potential issues with the delivery or rendering ads.
+
```

Comment:
perhaps helpful to note here that prebid sends hb_bidder_BIDDER and this is useless